iphone/metadata/de-DE/description.txt

```diff
@@ -0,0 +1,37 @@
+• Detaillierte Offline-Karten mit Orten, die auf anderen Karten nicht zu finden sind
```

Comment:
Can you please update the description here and in Android with this list? We'll also update it for the English version:

    Detailed offline maps with places that don't exist on other maps, thanks to OpenStreetMap.org
    Cycling routes, hiking trails, and walking paths
    Contour lines, elevation profiles, peaks, and slopes
    Turn-by-turn walking, cycling, and car navigation with voice guidance
    Fast offline search on the map
    Bookmarks export and import in KML/KMZ formats (GPX is planned)
    Dark Mode to protect your eyes
    Countries and regions don't take a lot of space
    Free and open-source